sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/FactoryIdentity.cs

```diff
@@ -31,31 +32,35 @@ public FactoryIdentity()
         /// <summary>
         /// Initializes a new instance of the FactoryIdentity class.
         /// </summary>
+        /// <param name="type">The identity type. Possible values include:
+        /// 'SystemAssigned', 'UserAssigned',
+        /// 'SystemAssigned,UserAssigned'</param>
         /// <param name="principalId">The principal id of the identity.</param>
         /// <param name="tenantId">The client tenant id of the
         /// identity.</param>
         /// <param name="userAssignedIdentities">List of user assigned
         /// identities for the factory.</param>
-        public FactoryIdentity(System.Guid? principalId = default(System.Guid?), System.Guid? tenantId = default(System.Guid?), IDictionary<string, object> userAssignedIdentities = default(IDictionary<string, object>))
+        public FactoryIdentity(string type, System.Guid? principalId = default(System.Guid?), System.Guid? tenantId = default(System.Guid?), IDictionary<string, object> userAssignedIdentities = default(IDictionary<string, object>))
```

Comment:
Breaking change - you should include a customization that implements the old method signature as an overload for this.  This is simply adding a partial class in the customizations folder with the old constructor calling the new constructor.

Reply:
Add class /src/Extensions/FactoryIdentity.cs